README.md

```diff
@@ -2,71 +2,21 @@
 
 This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
 
-## Setup
+----
 
-- Clone this repo! (And navigate into it)
-  ```
-  git clone https://github.com/thewca/worldcubeassociation.org
-  cd worldcubeassociation.org
-  ```
+## Common Queries
+- [Overview of the WCA's software ecosystem](https://docs.worldcubeassociation.org/)
+- [Running the website locally](https://docs.worldcubeassociation.org/contributing/quickstart)
+- [Contributing Guide](https://docs.worldcubeassociation.org/contributing/detailed_contributing_guide.html)
+- [API to read data from the website](https://wca-rest-api.robiningelbrecht.be/)
+- [Other WCA Repos](https://docs.worldcubeassociation.org/#wca-software-resources)
+- [Using OAuth, or writing data to the website](https://docs.worldcubeassociation.org/knowledge_base/v0_api.html)
+- [Joining WST]()
 
-We can run the application in two ways:
-- [Run using Docker](#run-using-docker)
-- [Run directly with Ruby](#run-directly-with-ruby-lightweight-also-only-runs-the-rails-portions-of-the-site)
+## Can I join WST?
+Currently (2023-09-10) WST is not accepting new members. We aim to onboard more volunteers by early 2024.
```

Comment:
winter 2023 :muscle:

Reply:
Do you mean I should replace the date with winter 2023, or did you mean "winter 2024"? In either case, I am very opposed to specifying a season given the fact that we're having this conversation across hemispheres :P But I think specifying a quarter can help. I've made it Q2 2024 to give us some breathing room

Reply:
Reading this again, do you mean we'd look to onboard volunteers by the end of the year? Happy to change it to that if so

Reply:
I think that we're in a position to onboard volunteers again around the year change 2023/2024